bin/cloud-local.sh

```diff
@@ -34,6 +34,18 @@ fi
 . "${bin}"/ports.sh
 
 function download_packages {
+  # Is the pre-download packages variable set?
+  if [[ ! -z ${pkg_pre_download+x} ]]; then
+    # Does that folder actually exist?
+    if [[ -d ${pkg_pre_download} ]] ; then
+      test -d ${CLOUD_HOME}/pkg || rmdir ${CLOUD_HOME}/pkg
+      test -h ${CLOUD_HOME}/pkg && rm ${CLOUD_HOME}/pkg
+      ln -s ${pkg_pre_download} ${CLOUD_HOME}/pkg
```

Comment:
one thing to think about here for symlink vs copy is what happens when the target is not read-only or 444 perms...if i did a git pull and smooshed my config and then tried to run things we might conflict there. 

Using a copy is better in that case and perhaps "safer"?